tests/short-first-segment.sh

```diff
@@ -0,0 +1,24 @@
+#! /bin/sh -e
+
+PATCHELF=$(readlink -f "../src/patchelf")
+SCRATCH="scratch/$(basename "$0" .sh)"
+READELF=${READELF:-readelf}
+
+EXEC_NAME="short-first-segment"
+
+if test "$(uname -i)" = x86_64 && test "$(uname)" = Linux; then
```

Comment:
Maybe the test should not fail if the package is not installed. gzip should be more mainstream on elf-based platforms than zip.

```suggestion
if test "$(uname -i)" = x86_64 && test "$(uname)" = Linux && gzip --version 2>/dev/null; then
```

Reply:
Fixed.